experimental/dds/tree2/src/test/shared-tree/fuzz/fuzzEditReducers.ts

```diff
@@ -50,7 +74,7 @@ export function checkTreesAreSynchronized(trees: readonly ISharedTree[]) {
 	}
 }
 
-function applyFieldEdit(tree: ISharedTree, fieldEdit: FieldEdit): void {
+function applyFieldEdit(tree: ISharedTree | ISharedTreeView, fieldEdit: FieldEdit): void {
```

Comment:
since `ISharedTree` extends `ISharedTreeView` and you don't appear to be doing any sort of narrowing in the implementation, it seems like all of these functions should maybe just take in `ISharedTreeView` rather than the union.